calico_containers/pycalico/ipam.py

```diff
@@ -112,12 +122,22 @@ def unassign_address(self, pool, address):
         Unassign an IP from a pool.
 
         :param IPPool or IPNetwork pool: The pool that the assignment is from.
+        If the pool is None, get the pool from datastore
         :param IPAddress address: The address to unassign.
-
         :return: True if the address was unassigned, false otherwise. An
         exception is thrown for any error conditions.
         :rtype: bool
         """
+        if pool is None:
```

Comment:
This functionality is duplicated once. Does it make sense to throw it in a function, and then 
```
pool = pool or get_pool(address)
```
?